### PR TITLE
Multi model option

### LINKED ISSUE #124: Create a multi model structure
Because the RCT will run parallel to the normal implementation, we need to create an infrastructure that selects the prediction model based on the hoofdagenda

### PR DESCRIPTION
**Pull request by WoltersEric**
closes #124

### CODE REVIEW

File: tests/test_noshow.py [hunk 1]
```diff
@@ -26,11 +31,10 @@ def fake_postal_codes(_=None) -> pd.DataFrame:
...
 
 
-def fake_model(_=None):
-    """Mock function for returning `load_model`
-
-    Needs to accept a parameter, but this parameter will be ignored
-    """
+def fake_model(model_path=None):
+    """Mock function for returning `load_model`"""
+    if model_path and "rct" in str(model_path):
```
Comment (rubenpeters91) [inline tests/test_noshow.py:36]: what if the model_path does not have rct in the name? Maybe throw an exception or something?

**Review by rubenpeters91 — COMMENTED** [review]
Looks good, maybe we should copy the current model to the rct model in this PR as well, so the code keeps working, while we wait for the new appointments?

File: tests/test_noshow.py (hunk 1 above)
Comment (WoltersEric) [reply tests/test_noshow.py:36]: If there isn't 'rct' in the model_path it should select the regular model. At least that how I see it

**Comment by WoltersEric** [comment]
Deployed new api to test environment which seems to work

**Pull request closed without merging**